docs/setup/README.md

```diff
@@ -53,6 +53,11 @@ npx firebase login
 # Select the project you've just created and use "default" as alias
 npx firebase use --add
 
+# Push your changes
+git add .firebaserc src/firebase/init.js
+git commit -m 'Add firebase config'
```

Comment:
It is ok for me ! Just change the commit message to be suitable with gitmoji. I have added a suggestion.